From bb46a8a729cc4d66ad36db40c17e36a5111f19c3 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Fri, 1 Oct 2021 13:00:24 +0200
Subject: [PATCH] Cargo.toml: do not abort on panic

OE's rust is configured to unwind, and this setting clashes with it/

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>

---
 Cargo.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index f576534bf3..5ecc17c319 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -56,13 +56,11 @@ opt-level = 1
 rpath = false
 lto = false
 debug-assertions = true
-panic = "abort"
 
 [profile.release]
 opt-level = 2
 rpath = false
 debug-assertions = false
-panic = "abort"
 
 # Optimize build dependencies, because bindgen and proc macros / style
 # compilation take more to run than to build otherwise.
